### PR TITLE
[refactor](cluster)(step-3) remove cluster related to Auth

### PR DESCRIPTION
**Pull request by morningman**
## Proposed changes

Issue Number: #19897

Remove `default_cluster` prefix related to:
1. User
2. Role
3. UserManager
4. RoleManager
5. UserRoleManager
6. UserProperty
7. Create/Drop user Stmt
8. Create/Drop role Stmt
9. Grant/Revoke

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

**Comment by morningman** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java [hunk 1]
```diff
@@ -266,11 +266,10 @@ private boolean parseAuthInfo(HttpServletRequest request, ActionAuthorizationInf
...
             final String[] elements = authInfo.fullUserName.split("@");
             if (elements != null && elements.length < 2) {
-                authInfo.fullUserName = ClusterNamespace.getFullName(SystemInfoService.DEFAULT_CLUSTER,
-                        authInfo.fullUserName);
+                authInfo.fullUserName = ClusterNamespace.getNameFromFullName(authInfo.fullUserName);
                 authInfo.cluster = SystemInfoService.DEFAULT_CLUSTER;
             } else if (elements != null && elements.length == 2) {
-                authInfo.fullUserName = ClusterNamespace.getFullName(elements[1], elements[0]);
+                authInfo.fullUserName = ClusterNamespace.getNameFromFullName(elements[0]);
                 authInfo.cluster = elements[1];
```
Comment (zddr) [inline fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java:273]: authInfo.cluster do not need remove?

**Review by zddr — COMMENTED** [review]
AccessControllerManager. checkDbPriv if need change?

public boolean checkDbPriv(UserIdentity currentUser, String ctl, String db, PrivPredicate wanted) {
        boolean hasGlobal = sysAccessController.checkGlobalPriv(currentUser, wanted);
        String qualifiedDb = ClusterNamespace.getFullName(SystemInfoService.DEFAULT_CLUSTER, db);
        return getAccessControllerOrDefault(ctl).checkDbPriv(hasGlobal, currentUser, ctl, qualifiedDb, wanted);
    }

File: fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java (hunk 1 above)
Comment (morningman) [reply fe/fe-core/src/main/java/org/apache/doris/httpv2/controller/BaseController.java:273]: No, I will remove this in next pr

**Review by zddr — APPROVED** [review]

**Review by yiguolei — APPROVED** [review]
LGTM

**Pull request merged**